utils/qemu/Makefile

```diff
@@ -417,6 +418,7 @@ CONFIGURE_ARGS +=			\
 	--disable-qom-cast-debug	\
 	--disable-rbd			\
 	--disable-rdma			\
+	--disable-rutabaga-gfx          \
```

Comment:
dtto

Reply:
And again, we haven't use new graphics by default. Also as i know that more useful for the cloud, than wrt like single node appilcations.

Reply:
@BKPepe also do you know that't is the sate of Rust libararies like libblkio or that rustabaga ones in the WRT?

Reply:
OpenWrt has rust toolchain, but these packages aren't added yet. You can create new Makefile for them.

Reply:
@1715173329 thanks for the info. I'm not going to use features that requires that libraries, so i'm not the best candidate to maintain those packages.